utils/utils.go

```
@@ -172,3 +174,13 @@ func IsPublicNetwork(networkID uint32) bool {
 func IsCustomNetwork(networkID uint32) bool {
 	return !IsPublicNetwork(networkID) && networkID != constants.LocalID
 }
+
+func BeaconMapToSet(beaconMap map[ids.NodeID]netip.AddrPort) beacon.Set {
```

please do not ignore errors